apps/wing/src/analytics/scripts/detached-export.ts

```diff
@@ -0,0 +1,30 @@
+import { getAnonymousId, loadAnalyticsEvent } from '../storage';
+import * as fs from 'fs';
+import Analytics from '@segment/analytics-node';
+import { getWingAnalyticsCollectionConfig } from '../collect';
+
+// When this file is run as a child process, it will be passed the path 
+// to the analytics report file
+const filePath = process.argv[2];
+
+try {
+  const analytics = new Analytics({ writeKey: 'sCqPF5xSscOjJdi5Tbkqu73vfF8zkZdw'});
+  const event = loadAnalyticsEvent(filePath);
+  const analyticsConfig = getWingAnalyticsCollectionConfig();
+
+  if (analyticsConfig.export) {
+    analytics.track({
+      anonymousId: getAnonymousId(),
+      timestamp: event.timestamp,
+      event: event.event,
+      properties: event.properties,
+    });
+  }
+
+  // Keeps the file around for debugging purposes
+  if (!analyticsConfig.debug) {
+    fs.unlinkSync(filePath);
+  }
+} catch(err) {
+  // TODO: add mechanism to retry
```

Comment:
i am assuming if we fail, we will simply send the file again next time, no? that seems like a reasonable retry

Reply:
currently in order to avoid a race condition (if multiple wing commands run in parallel)  the detached script is called with a specific analytic file uuid, so there would need to be some sort of watch or failure log or something for the subsequent runs to know to try exporting the failed metric.

Reply:
imo I don't think we should burden the user's system/io with retries. Metrics are great but let's not be greedy, especially since our metrics are opt-out. 
At most we should retry once in this same process (which luckily doesn't block anything) and if it doesn't work then just consider the data lost.